react-app/src/features/login/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { AwsCognitoOAuthOpts } from "@aws-amplify/auth/lib-esm/types/Auth";
+import { Auth } from "aws-amplify";
+import { Link } from "react-router";
+
+import config from "@/config";
+import { FAQ_TAB } from "@/consts";
+import { cn } from "@/utils";
+
+const buttonStyling =
+  "inline-flex p-2 items-center justify-center rounded-md text-[16px] font-bold ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50";
```

Comment:
Just noting that if we ever use `Link` buttons again, we should make a new component